desktop/cmp/persistenceManager/PersistenceManager.ts

```diff
@@ -0,0 +1,152 @@
+import {div, fragment, hbox} from '@xh/hoist/cmp/layout';
+import {hoistCmp, HoistProps, uses} from '@xh/hoist/core';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {TreeView} from '@xh/hoist/desktop/cmp/persistenceManager/Types';
+import {Icon} from '@xh/hoist/icon/Icon';
+import {menu, menuDivider, menuItem, popover} from '@xh/hoist/kit/blueprint';
+import {capitalize} from 'lodash';
+import {ReactNode} from 'react';
+import {manageDialog} from './impl/ManageDialog';
+import {saveDialog} from './impl/SaveDialog';
+import './PersistenceManager.scss';
+import {PersistenceManagerModel} from './PersistenceManagerModel';
+import {pluralize} from '@xh/hoist/utils/js';
+
+interface PersistenceManagerProps extends HoistProps<PersistenceManagerModel> {
+    /** True to disable options for saving/managing items. */
+    minimal?: boolean;
+}
+
+export const [PersistenceManager, persistenceManager] =
+    hoistCmp.withFactory<PersistenceManagerProps>({
+        displayName: 'PersistenceManager',
+        model: uses(PersistenceManagerModel),
+
+        render({model, minimal = false}) {
+            const {
```

Comment:
Something to think about - should we mask on the initial load? (i.e. while getting the list of views before we load the selected view).  There will inevitably be some jumpiness here.  Ideally we'd want to mask the components for which we're controlling state so the user can't nudge something before the selected view is loaded.